app/views/namespaces/components/_form.html.slim

```diff
@@ -9,10 +9,9 @@ div ref="form" class="collapse"
         span.help-block
           span v-if="!$v.namespace.name.required"
             | Name can't be blank
-          span v-if="!$v.namespace.name.format"
-            | Name can only contain lower case alphanumeric characters, with optional underscores and dashes in the middle
-          span v-if="!$v.namespace.name.available"
-            | Name has already been taken
+          span.error-messages v-if="errors.name"
```

Comment:
Based on the comment I just left on about setting the the errors messages, I think we can change this condition to `errors.name.length` to avoid the rendering of an empty `span.error-messages`.